### PR TITLE
s4706843 pull request

### PR DESCRIPTION
**Pull request by tiankong-sky**
This is my GFNetPyramid model, which will classify brain images from the ADNI dataset. The results are printed and/or saved when running the training. The best accuracy I achieved on the test set was 65%. Almost 100% accuracy was achieved on the training set. The overfitting problem was not fully resolved due to time issues. But the model is fully functional.

### CODE REVIEW

**Comment by aniketgupta17** [comment]
Observational Feedback

Pull Request:
Correctly created the Pull request from Topic Recognition Branch .
The pull request includes NO description about the file structure .
No previous feedback incorporated . 

File Organizing: Well-organized files.

Commit Log:
Commit messages should be more informative 
Commits are  not regularly made.

Documentation:
The README file **should** contain in depth  explanation of the algorithm and the problem it solves.
Code comments and docstrings  are not included.
Proper GitHub markdown formatting is used .
Can add about Conclusion and  future improvements in the ReadMe.

**Comment by gayanku** [comment]
## Marking
<table><tbody>
  <tr><td colspan='3'><b>Good/OK/Fair Practice (Design/Commenting, TF/Torch Usage)</b></td> </tr>
	<tr><td></td><td>Good design and implementation. </td><td></td></tr>
	<tr><td></td><td>Spacing and comments. </td><td></td></tr>
	<tr><td></td><td>No Header blocks. </td><td>-1</td></tr>
	
  <tr><td colspan='3'><b>Recognition Problem</b></td></tr>
	<tr><td></td><td>OK solution to problem. Acc low, no validation.</td><td>-2</td></tr>
	<tr><td></td><td>Driver Script present. </td><td></td></tr>
	<tr><td></td><td>File structure present. </td><td></td></tr>
	<tr><td></td><td>Good Usage & Demo & Visualisation & Data usage. </td><td></td></tr>
	<tr><td></td><td>Module present. </td><td></td></tr>
	<tr><td></td><td>Commenting missing. </td><td>-1</td></tr>
	<tr><td></td><td>Data leakage found. No Data Aug. No data splits.</td><td>-2</td></tr>
	<tr><td></td><td>Difficulty : Hard.  GFNet (Hard Difficulty)</td><td></td></tr>
	
  <tr><td colspan='3'><b>Commit Log</b></td></tr>
	<tr><td></td><td>Some/Adequate Meaningful commit messages. </td><td>-1</td></tr>
	<tr><td></td><td>Good Progressive commits. </td><td>-2</td></tr>
	
  <tr><td colspan='3'><b>Documentation</b></td></tr>
	<tr><td></td><td>Readme :Acceptable. </td><td>-1</td></tr>
	<tr><td></td><td>Model/technical explanation :Acceptable. </td><td>-1</td></tr>
	<tr><td></td><td>Description and Comments :Acceptable. </td><td>-1</td></tr>
	<tr><td></td><td>Markdown used and PDF submitted. PDF Checked.</td><td></td></tr>
	
  <tr><td colspan='3'><b>Pull Request</b></td></tr>
	<tr><td></td><td>Successful Pull Request (Working Algorithm Delivered on Time in Correct Branch). </td><td></td></tr>
	<tr><td></td><td>No Feedback required. </td><td></td></tr>
	<tr><td></td><td>Request Description is adequate. </td><td>-1</td></tr>
  <tr><td colspan='2'><b>TOTAL</b></td><td>-13</td></tr>
</tbody></table>

Marked as per the due date and changes after which aren't necessarily allowed to contribute to grade for fairness.
_Subject to approval from Shakes_

**Comment by hanemma7moud** [comment]
Tianyu Wang

**Comment by shakes76** [comment]
Cant merge because of conflicting changes to main repo files (gitignore). Please update for merge, doesn't affect grade.
